pkg/gitprovider/azure/azure.go

```diff
@@ -181,6 +181,79 @@ func (p *provider) ListPullRequests(
 	return pts, nil
 }
 
+// MergePullRequest implements gitprovider.Interface.
+func (p *provider) MergePullRequest(
+	ctx context.Context,
+	id int64,
+) (*gitprovider.PullRequest, bool, error) {
+	gitClient, err := adogit.NewClient(ctx, p.connection)
+	if err != nil {
+		return nil, false, fmt.Errorf("error creating Azure DevOps client: %w", err)
+	}
+
+	// Get the current PR to check its status and get the last merge source commit
+	currentPR, err := gitClient.GetPullRequest(ctx, adogit.GetPullRequestArgs{
```

Comment:
"current" is an odd word to be using here because that's a temporal or relative term. There's no previous PR. There's no next PR. Isn't this just _the_ PR?

Reply:
we are assigning PR at multiple places across the codebase, hence I will use `adoPR` for the Azure DevOps pull request object (the native type) and use `pr` only for the final converted `gitprovider.PullRequest` that gets returned. i think this is cleaner than using "current"